Helper/Config.php

```diff
@@ -292,4 +292,29 @@ public function initSdkConfiguration(bool $isAuthInfoRequired = true) : void
         SdkConfigManager::setValue(sprintf('%s.authorization.auth_user', $sdkEnv), $this->getAuthenticatedUserName());
         SdkConfigManager::setValue(sprintf('%s.authorization.access_token', $sdkEnv), (string) $this->getAccessToken());
     }
+
+    /**
+     * Retrieve a list of stores that the user has access to
+     *
+     * @return $stores array a list of stores
+     */
+    public function getStores()
+    {
+
+        $om = $this->context->getObjectManager();
+        $storeManager = $om->get('Magento\Store\Model\StoreManagerInterface');
+        $storeManagerDataList = $storeManager->getStores($withDefault = false);
+        $store = array();
```

Comment:
`stores`?

Reply:
Additionally, I think the Magento2 coding standards (the linter we use in this case) will complain about `array()` instead of `[]`

Reply:
also fixed